CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,68 @@
+# Contributors' Guide
+
+This guide will help familiarize contributors to the `containerd/containerd` repository.
+
+## Prerequisite
+
+First read the containerd project's [general guidelines around contribution](https://github.com/containerd/project/blob/main/CONTRIBUTING.md)
+which apply to all containerd projects.
+
+## Getting Started
+
+See [`BUILDING.md`](https://github.com/containerd/containerd/blob/main/BUILDING.md) to setup a local development environment.
+
+If you are also a new user to containerd, you can first check out the [_Getting starting with containerd_](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) guide.
+
+## Setting up your environment
+
+At a minimum, the dev tools from `script/setup/install-dev-tools` should be installed.
+Other install scripts under `script/setup` may need to be run depending on your environment and your preference for installing libraries and dependencies.
```

Comment:
see make install-deps .. hmm definitely two patterns depending on preference

Reply:
We should try and have one way to do it. I'll mention `make install-deps`, but is for installing the CRI dependencies today which makes no sense really.

Reply:
more for the people working on Kubernetes/Rancher/... features, running local clusters and/or using crictl/critest :)